stages/org.osbuild.ostree.genkey

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/python3
+"""
+Generate ed25519 public/private keypair in format used by ostree.
```

Comment:
..."to bind together the initramfs and root for composefs, in a manner similar to kernel modules" or so?   Just "used by ostree" isn't very explanatory.

And let's link to the ostree composefs docs.

Reply:
Well, technically this is just the generic "ostree sign" operation, so it has other uses too. However, you're right, we should mention the expected use case more prominently.

Reply:
OK backing up a second though...maybe this stage could land as `org.osbuild.gentransientkey` or something?  There's nothing ostree specific about it actually.

Reply:
I'd prefer to keep it under `ostree`, the splitting out of the PEM is likely not relevant to everything?